### ISSUE TITLE
Link to CSPA?

### ISSUE THREAD

**Issue by RLesur** [ThomasFaria/retex-innovation-insee#12]
Last November, @trygu [has given a talk](https://unece.org/sites/default/files/2023-11/HLG2023_Soapbox7_Onyxia%20and%20the%20dream%20of%20CSPA.pdf) stating that Onyxia was the first (big) step towards [CSPA](https://statswiki.unece.org/display/CSPA).  
TBH, I'd never heard of CSPA before, as the ideas date back a few years.  
I wonder if it's useful/interesting to repeat Trygve's thoughts in this paper.

Perhaps @trygu could give us his opinion and make a proposal for an addition?

**Comment by avouacr** [comment]
Yeah I won't take the risk of including it myself as I'm not knowledgeable enough on the matter, but indeed any proposal welcome @trygu if you think this would be a relevant addition
